README.md

```diff
@@ -62,6 +62,10 @@ Ethereum node versions currently tested and supported:
 - [Parity 1.11+](https://github.com/paritytech/parity-ethereum/releases) (due to a [fix with pubsub](https://github.com/paritytech/parity/issues/6590).)
 - [Geth 1.8+](https://github.com/ethereum/go-ethereum/releases)
 
+### Create an environment file
+Create an `.env` file and populate with variables specific to the network you're running on.
```

Comment:
Typically I've seen projects offer advice such as `cp .env.example .env` ... do we have an example?

Reply:
Not atm, but I'm think this could be a candidate
```
ETH_URL=$ETH_URL
ETH_CHAIN_ID=$ETH_CHAIN_ID
LINK_CONTRACT_ADDRESS=$LINK_CONTRACT_ADDRESS
DATABASE_URL=postgresql://$USERNAME:$PASSWORD@$SERVER:$PORT/$DATABASE

LOG_LEVEL=debug
MIN_OUTGOING_CONFIRMATIONS=2
CHAINLINK_TLS_PORT=0
SECURE_COOKIES=false
GAS_UPDATER_ENABLED=true
ALLOW_ORIGINS=*
```